shuup/gdpr/static_src/gdpr.js

```diff
@@ -55,4 +58,13 @@
     $("#agree-btn").click(() => {
         saveConsent();
     });
+    }
+
+    if(typeof jQuery === "undefined"){
```

Comment:
format this block correctly
![image](https://user-images.githubusercontent.com/8770370/45319078-2a18a900-b515-11e8-8a92-c5c7da531c6a.png)